ext/junit/java/androidx/test/ext/junit/rules/ActivityScenarioRule.java

```diff
@@ -86,25 +105,43 @@ public ActivityScenarioRule(Class<A> activityClass, @Nullable Bundle activityOpt
    */
   public ActivityScenarioRule(Intent startActivityIntent) {
     scenarioSupplier = () -> ActivityScenario.launch(checkNotNull(startActivityIntent));
+    this.launchActivity = true;
   }
 
   /**
    * @see #ActivityScenarioRule(Intent)
    * @param activityOptions an activity options bundle to be passed along with the intent to start
-   *     activity.
+   *                        activity.
    */
   public ActivityScenarioRule(Intent startActivityIntent, @Nullable Bundle activityOptions) {
     scenarioSupplier =
         () -> ActivityScenario.launch(checkNotNull(startActivityIntent), activityOptions);
+    this.launchActivity = true;
+  }
+
+  /**
+   * @see #ActivityScenarioRule(Intent)
+   * @param activityOptions an activity options bundle to be passed along with the intent to start
+   *                        activity.
+   * @param launchActivity true if the Activity should be launched automatically once per test. If
+   *                       set to false the launch of the activity under test will be deferred until
+   *                      {@link ActivityScenarioRule#getScenario()} is called.
+   */
+  public ActivityScenarioRule(Intent startActivityIntent, @Nullable Bundle activityOptions, boolean launchActivity) {
+    scenarioSupplier =
+            () -> ActivityScenario.launch(checkNotNull(startActivityIntent), activityOptions);
+    this.launchActivity = launchActivity;
   }
 
   @Override
-  protected void before() throws Throwable {
+  protected void before() {
+    if (!launchActivity) return;
     scenario = scenarioSupplier.get();
   }
 
   @Override
   protected void after() {
+    if (scenario == null) return;
     scenario.close();
   }
```

Comment:
Minor remark: Instead of an early return it might be better to throw:

```java
@Override
protected void after() {
  if (!launchActivity && scenario == null) {
    throw new ActivityScenarioRuleException("Did you remember to call getScenario() to launch the activity under test.");
  }
  scenario.close();
}
```

It can be left as it is though and then it will be caught by `RootViewPicker`:
```
java.lang.RuntimeException: No activities found. Did you forget to launch the activity by calling getActivity() or startActivitySync or similar?
at androidx.test.espresso.base.RootViewPicker.waitForAtLeastOneActivityToBeResumed(RootViewPicker.java:52)
at androidx.test.espresso.base.RootViewPicker.get(RootViewPicker.java:15)
...
```

Using a test similar to this:
```java
@Test
fun testNoLaunch() {
    onView(withId(R.id.fragment_recyclerview)).check(hasItemsCount(items.size))
}
```